components/x-podcast-launchers/src/copy-to-clipboard.js

```diff
@@ -0,0 +1,25 @@
+export default function  copyToClipboard (event) {
+	const inputEl = event.target.parentElement.querySelector('input');
+	const oldContentEditable = inputEl.contentEditable;
+	const oldReadOnly = inputEl.readOnly;
+	const range = document.createRange();
+
+	inputEl.contenteditable = true;
+	inputEl.readonly = false;
+	inputEl.focus();
+	range.selectNodeContents(inputEl);
+
+	const selection = window.getSelection();
+
+	try {
+		selection.removeAllRanges();
+		selection.addRange(range);
+		inputEl.setSelectionRange(0, 999999);
+	} catch (err) {
+		inputEl.select(); // IE11 etc.
+	}
+	inputEl.contentEditable = oldContentEditable;
+	inputEl.readOnly = oldReadOnly;
+	document.execCommand('copy');
+	inputEl.blur();
```

Comment:
where does the cursor end up after you've done this? is there a way to avoid moving the focus on to the element and then blurring again? that way the user's cursor stays where it is

we've another example of copy code here: https://github.com/Financial-Times/next-myft-page/blob/d106acb16df3b7b2f6991c0cf2426c3fc7e7900e/client/components/alerts/js/rss-preference.js